src/controllers/rankingsController.ts

```
@@ -1,11 +1,22 @@
 import { Request, Response } from 'express';
 import axios from 'axios';
 
+const RANKINGS_S3_URL = 'https://sci-temporary-bucket.s3.us-west-2.amazonaws.com/rankings.json';
+
 const rankings = async (req: Request, res: Response): Promise<void> => {
   try {
-    const rankingsS3DataUrl = 'https://sci-temporary-bucket.s3.us-west-2.amazonaws.com/rankings.json';
-    const response = await axios.get(rankingsS3DataUrl);
-    res.json(response.data);
+
+    const eventId = (req.query.eventId as string) || "333";
+    const stateId = (req.query.stateId as string) || "MH";
+    const type = (req.query.type as string) || "single";
```

Instead of setting default values, can we throw 400 error if any of the parameter is missing?

Something like:

```
const {eventId, stateId, type} = req.query;
if (!eventId || !stateId || !type) {
  return res.status(400).json({ error: "Missing required parameters." });
}
```

---

Done